### PR TITLE
chore: updated readme with testing integration section

### PR DESCRIPTION
**Pull request by TMaszko**
### Summary

This PR adds section about testing integration on devices/simulators. It could be useful especially if i.e someone doesn't have both iOS and Android devices.

### CODE REVIEW

**Review by mrousavy — APPROVED** [review]
The facebook steps are simply adding the ID in the "test users" list, right? In that case a simple link is enough, looks good 👍

File: README.md [hunk 1]
```diff
@@ -105,6 +105,20 @@ Follow [Facebook's instructions](https://www.facebook.com/help/publisher/1195459
...
+
+Follow [Facebook's instructions](https://developers.facebook.com/docs/audience-network/guides/test) to add test devices and add test users.
+
+#### Android
+
+You can get AAID from the android device/emulator by going to **Settings > Google > Ads**
+
+#### iOS
+
+You can get IDFA from the iOS device using third party tool. There is a wide selection of apps with the capability on the App Store. You can get IDFA directly from the iOS simulator by running this command: `xcrun simctl list 'devices' 'booted'`.
```
Comment (thymikee) [inline README.md:118]: Don't think we need to mention that?
```suggestion
You can get IDFA from the iOS device using third party tool. You can get IDFA directly from the iOS simulator by running this command: `xcrun simctl list 'devices' 'booted'`.
```

**Review by thymikee — APPROVED** [review]
👍🏼

File: README.md (hunk 1 above)
Comment (TMaszko) [reply README.md:118]: Thanks ! :)

**Pull request merged**